perma_web/perma/templates/user_management/create-link.html

```diff
@@ -11,6 +11,15 @@
 
 {% block mainContent %}
 
+ <!-- vue app -->
+ {% if request.user.is_staff %}
+  {% flag "vue-dashboard" %}
+  <!-- <div class="vanilla-div">Vanilla JavaScript Count: None</div>
+  <button class="vanilla-button">Increment Counter</button> -->
+    <div id="vue-app"></div>
+  {% endflag %}
+{% endif %}
+
```

Comment:
I moved this up so that I didn't need to keep scrolling to continually test the UI.